.authors.yml

```diff
@@ -597,17 +618,14 @@
   github: kalefranz
   alternate_emails:
   - kalefranz@users.noreply.github.com
-- name: distro-bot@anaconda.com
-  email: Distro Bot
-  num_commits: 1
-  first_commit: &id001 2021-11-11 14:06:14.526569
-  github: msarahan
 - name: Jacob Hultman
-  email: jhultman@novateurresearch.comgit config --global user.email jhultman@novateurresearch.com
+  email: jhultman@novateurresearch.com
   num_commits: 1
   first_commit: 2019-11-15 18:16:26
   aliases:
   - Jacob
+  alternate_emails:
+  - jhultman@novateurresearch.comgit config --global user.email jhultman@novateurresearch.com
```

Comment:
This `alternate_emails` section needs to be removed

Reply:
This is needed to correct the bad email used in the git history.